cpp/src/IceSSL/SChannelTransceiverI.cpp

```diff
@@ -45,6 +45,9 @@ namespace
             case SP_PROT_TLS1_2_CLIENT:
             case SP_PROT_TLS1_2_SERVER:
                 return "TLS 1.2";
+            case SP_PROT_TLS1_3_CLIENT:
```

Comment:
We probably (in the future) should remove all of the SSL and TLS < 1.2 stuff

Reply:
This is only used for the string conversions.